outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "webapp_url" {
+  value = "https://${azurerm_linux_web_app.main.default_hostname}"
+}
+
+output "cd_webhook" {
+  value = "https://${azurerm_linux_web_app.main.site_credential[0].name}:${azurerm_linux_web_app.main.site_credential[0].password}@${azurerm_linux_web_app.main.name}.scm.azurewebsites.net/docker/hook"
```

Comment:
Worth noting this has a password in it, so adding a `sensitive` tag to it (like you have for the inputs) would be wise